src/services/balancer/subgraph/entities/pools/index.ts

```diff
@@ -151,6 +194,43 @@ export default class Pools {
     return Promise.all(promises);
   }
 
+  private async getExcludedAddresses() {
+    try {
+      const { data } = await axios.get<ExcludedPoolAddressesResponse>(
+        'https://raw.githubusercontent.com/balancer-labs/bal-mining-scripts/master/config/exclude.json'
```

Comment:
@markusbkoch are you happy for me to use this URL?

Reply:
Yes. I've also pushed this with mock data, excluding some whales from a couple of pools on each network for testing purposes if you want to point to it for now
https://raw.githubusercontent.com/balancer-labs/bal-mining-scripts/frontend-exclude-test/config/exclude.json